### ISSUE TITLE
Cartilaginous malacia

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#7024]
**Preferred term label:**
Cartilaginous malacia

**Synonyms**
Cartilaginous tracheobronchomalacia

**Definition (free text, please give PubMed ID)**
Softening of the tracheal and/or bronchial cartilage resulting in segmental tracheal and/or bronchial weakness

**Parent term (use hpo.jax.org/app)**
Tracheobronchomalacia HP:0002786

**Diseases characterized by this term? (e.g. Orphanet or OMIM number)**
Relapsing Polychondritis ORPHA:728
Mounier-Kuhn syndrome OMIM 275300

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
@eswietlik  How is this different from the parent term ``Tracheobronchomalacia``? Doesn't this always involve the cartilage?

**Comment by pnrobinson** [comment]
@eswietlik  is this different from https://hpo.jax.org/app/browse/term/HP:0002786 ?

**Comment by eswietlik** [comment]
Hi Peter,

Sorry for the late reply but I had my viva and then took some time off.
Not sure if the term in question has been removed but the correct term
should be:

Tracheobronchomalacia (exaggerated luminal tracheobronchial narrowing
during expiration and widening during inspiration (narrowing of tracheal or
bronchial lumen > 70% on expiration during bronchoscopic visualisation)
 -  Excessive dynamic airway collapse, synonym membranous malacia -
excessive anterior displacement of the membranous wall
 -  Cartilaginous tracheobronchomalacia (or just cartilaginous malacia) -
softening of the cartilage resulting in excessive narrowing of tracheal or
bronchial lumen.


The difference between the two subtypes is membranous malacia vs
cartilaginous malacia

Many thanks,
Emilia

   -


czw., 15 lip 2021 o 13:01 Peter Robinson ***@***.***>
napisał(a):

**Comment by pnrobinson** [comment]
@eswietlik  Thanks, got it, I added the term.
